data_structures/src/serializers.rs

```diff
@@ -32,6 +34,18 @@ use flatbuffers::FlatBufferBuilder;
 
 const FTB_SIZE: usize = 1024;
 
+#[derive(Debug, Clone, Copy)]
+struct GetBlocksFlatbufferArgs {
```

Comment:
I think that we don't need 2 structs that are the same (i.e. `GetBlocksFlatbufferArgs` and `GetBlocksWitnetArgs`).
Unfortunately, the same happened with other commands (e.g. `PingFlatbufferArgs`, ...). :disappointed: